source/includes/indexes/indexes.py

```diff
@@ -88,26 +88,56 @@
 collection.create_search_index(index)
 # end-create-search-index
 
+# start-create-vector-search-index
+
+from pymongo.operations import SearchIndexModel
+
+search_index_model = SearchIndexModel(
+  definition={
+    "fields": [
+      {
+        "type": "vector",
+        "numDimensions": <numberOfDimensions>,
```

Comment:
S: check if using camelcase is the convention here 
otherwise you can use something like `number_of_dims`